### ISSUE TITLE
Custom Taxonomies assigned as Top Level Menu items are not usable by Editors

### ISSUE THREAD

**Issue by PodsBot** [pods-framework/pods#3569]
Custom Taxonomies assigned as Top Level Menu items are not usable by Editors submitted via Slack by jimtrue

**Comment by acmario** [comment]
The custom taxonomies are still accessible by Editors as a submenu of the assigned CPT but not as a top level menu (usable only by Administrators)

**Comment by JoryHogeveen** [comment]
@acmario 
Sorry it took so long but there is a capability for this.
`pods_edit_**PODNAME**`

Example: Taxonomy "product_groups" would be `pods_edit_productgroups`.

The reason for this is because these type of menu locations are not added through the default WP functions and thus do not validate the default WP capabilities.
See https://github.com/pods-framework/pods/blob/2.x/classes/PodsAdmin.php#L327

@sc0ttkclark Maybe a good idea to validate for the taxonomy capabilities from WP since these are now fully sported since 2.6.7

**Comment by acmario** [comment]
I can confirm that setting the custom capability (pods_edit_**PODNAME**) for the desidered rule solve the problem. Thank you!

**Comment by JoryHogeveen** [comment]
No problem! Please note that if #3780 gets merged this will revert to the base WP capabilities.

Default:
`manage_categories`

Custom (if you've enabled this in Pods)
`manage_**YOUR_CAPABILITY**_terms`
